src/containers/org/roles/EditRoleDetailsBody.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+import React, { useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+import {
+  Button,
+  Input,
+  Typography,
+} from 'lattice-ui-kit';
+import { useDispatch } from 'react-redux';
+import { RequestStates } from 'redux-reqseq';
+import type { Organization, Role } from 'lattice';
+import type { RequestState } from 'redux-reqseq';
+
+import { ModalBody, StackGrid } from '../../../components';
+import { resetRequestState } from '../../../core/redux/actions';
+import { EDIT_ROLE_DETAILS, editRoleDetails } from '../actions';
+
+const StyledBody = styled(ModalBody)`
+  padding-bottom: 30px;
+`;
+
+const StyledStack = styled(StackGrid)`
+  margin: 16px 0 32px 0;
+`;
```

Comment:
If I were doing it, I would have done this:

```
<StackGrid gap={24}>
  <StackGrid>
    ...
  </StackGrid>
  <Button>Save</Button>
</StackGrid>
```

However, this approach means an extra DOM node, which you avoid by using margins 🤔 . Anyway, no change required, just thinking out loud.